examples/file_prompt_all_types_guard_input.py

```diff
@@ -9,7 +10,8 @@
     # not required at all if set in environment variables
     security=acuvity.Security(
        token=os.getenv("ACUVITY_TOKEN", ""),
-    )
+    ),
+    client=httpx.Client(verify=False)
```

Comment:
no, never create a client which disables TLS verification in examples

Reply:
SDK is broken on dev so was testing on local host, but I pushed this also. Will need to revert this.